__tests__/app.test.js

```diff
@@ -111,8 +111,49 @@ describe("/api/articles", () => {
           votes: 100,
           article_img_url:
             "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
-          comment_count: 11
+          comment_count: 11,
         });
       });
   });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+  test("GET 200: should return array of comments for given article id sorted by most recent comments first", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const comments = body.comments;
+        comments.forEach((comment) => {
```

Comment:
As you're asserting in a forEach, could you check the length of the array beforehand please? 👍

Reply:
added in latest commit